ludwig/utils/data_utils.py

```diff
@@ -197,6 +197,10 @@ def read_stata(data_fp, df_lib):
     return df_lib.read_stata(data_fp)
 
 
+def read_hdf5(data_fp, df_lib):
+    return load_hdf5(data_fp, clean_cols=True)
```

Comment:
Why do we need `df_lib`? What does `clean_cols` provide? Asking because if both are not useful we could get away with using `load_hdf5`` directly.